flink-ml-core/src/test/java/org/apache/flink/ml/api/GraphTest.java

```diff
@@ -49,13 +46,7 @@ public class GraphTest extends AbstractTestBase {
 
     @Before
     public void before() {
-        Configuration config = new Configuration();
-        config.set(ExecutionCheckpointingOptions.ENABLE_CHECKPOINTS_AFTER_TASKS_FINISH, true);
-        env = StreamExecutionEnvironment.getExecutionEnvironment(config);
-        env.getConfig().enableObjectReuse();
-        env.setParallelism(4);
-        env.enableCheckpointing(100);
-        env.setRestartStrategy(RestartStrategies.noRestart());
+        env = org.apache.flink.ml.util.TestUtils.getExecutionEnvironment();
```

Comment:
There may be some ambiguity here. The `org.apache.flink.ml.util.TestUtils` here is under `flink-ml-core`. We also have a `org.apache.flink.ml.util.TestUtils` under `flink-ml-lib`, and both of them contains a `getExecutionEnvironment` method. This cause some code duplicates but avoid introduce test-jar of `flink-ml-core` in `flink-ml-lib`. 

I'm combining the `TestUtil` in `org.apache.flink.ml.api` and `org.apache.flink.ml.util` under `flink-ml-core` to avoid using full package path.

Reply:
According to the offline discussion, we agreed to merge the `TestUtils` in `flink-ml-lib` to `flink-ml-core`, and make `flink-ml-lib` depends on the test-jar of `flink-ml-core` to reduce code duplicates.